src/ai/openvino_model.py

```diff
@@ -0,0 +1,58 @@
+import openvino.properties as props
+from huggingface_hub import snapshot_download
+import openvino as ov
+import torch
+import os
+
+core = ov.Core()
+core.set_property({props.cache_dir: "cache"})
+
+__model_folder__ = os.path.join(
+    os.path.dirname(os.path.abspath(__file__)), "..", "..", "model"
+)
+
+
+class OVModel:
```

Comment:
I Don't see much logging in this class, is because we rely on underlying layer logging?

Reply:
I do not think loggin here is of any significance. Logging is captured above and below in the hierarchy